vscode/vscode.md

```diff
@@ -0,0 +1,93 @@
+This section outlines how to set up the application in the Visual Studio Code.
+
+## Prerequisites
+
+* Java SE 8, Java SE 11 or Java SE 17 is required.
+* Payara Server 5 is required. You can download Payara Server 5 from [here](https://www.payara.fish/downloads/).
+* The Visual Studio Code is required. You can download the Visual Studio Code from [here](https://code.visualstudio.com/download).
+* [Java Extension Pack, by Microsoft](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack)
+
+## Download
+
+[Download](https://github.com/eclipse-ee4j/cargotracker/archive/master.zip) the source code zip file and expand it somewhere in your file system. Note that this is a Maven project.
+
+## Visual Studio Code Setup
+
+You need to install the payara extensions from the Visual Studio Code marketplace :
+
+* [Payara Tools, by Payara](https://marketplace.visualstudio.com/items?itemName=Payara.payara-vscode)
+
+You can install extensions from Extensions view (Ctrl + Shift + X) in the activity bar of Visual Studio Code:
+
+![Payara](image/payara_server.png)
+ 
+Let’s start:
+
+ * Open the command pallet using Ctrl + Shift + P, type Maven and import the existing Maven Project :
+
+  ![Maven](image/maven.png)
+
+ * Or in the explorer sidebar expand Maven Projects, Select the + sign to create a Maven Project.
+
+ * Select the destination folder, where the project folder will be generated and answer the prompts (groupId, artifactId, version etc) in the focused TERMINAL window to setup the maven project.

```

Comment:
Why is this still here? This would be applicable if we are creating a new Maven project, which we are not.